dev/com.ibm.ws.channelfw/bnd.bnd

```diff
@@ -41,6 +41,7 @@ Import-Package: \
 	${defaultPackageImport}
 
 Private-Package: com.ibm.io.async, \
+    com.ibm.ws.channelfw.internal.chains;-split-package:=merge-first, \
```

Comment:
is this needed?

Reply:
This was interesting, I had put that line originally in the Import-Package stanza (with split-package provide) - thinking that it would be needed by the channelfw project (the package is split between cf and endpoint) but doing that gives problems with not being able to load 'local' members of the package such as Chain. So I moved it to 'Private-Package' in order to specify the 'split' and that worked.  BUT, in response to your comment,  I have just deleted it altogether and, to my surprise, it still works with the FATS I have tried. So - no. Will test full FAT run too.